modules/SDL.lua

```diff
@@ -1,15 +1,67 @@
 require('os')
 local sdl_logger = require('sdl_logger')
 local config = require('config')
+local console = require('console')
 local SDL = { }
 
 require('atf.util')
 
+SDL.buildOptions = {}
 SDL.exitOnCrash = true
 SDL.STOPPED = 0
 SDL.RUNNING = 1
 SDL.CRASH = -1
 
+local usedBuildOptions = {
+  remoteControl =  {
+    sdlBuildParameter = "REMOTE_CONTROL",
+    defaultValue = "ON"
+  },
+  extendedPolicy =  {
+    sdlBuildParameter = "EXTENDED_POLICY",
+    defaultValue = "PROPRIETARY"
+  }
+}
+
+local function readParameterFromCMakeCacheFile(paramName)
+  local pathToFile = config.pathToSDL .. "/CMakeCache.txt"
+  if is_file_exists(pathToFile) then
+    local paramValue, paramType
+    for line in io.lines(pathToFile) do
+      paramType, paramValue = string.match(line, "^%s*" .. paramName .. ":(.+)=(%S*)")
+      if paramValue then
+        return paramValue, paramType
+      end
+    end
+  end
+  return nil
+end
+
+local function setSdlBuildOption(optionName, sdlBuildParam, defaultValue)
```

Comment:
@aderiabin lack of description

Reply:
@LuxoftAKutsan  Description in LDoc format was added (see 5aec3b3)